sql/helpers.py

```diff
@@ -20,6 +29,8 @@ def wrapper(self, *args, **kwargs):
         try:
             return func(self, *args, **kwargs)
         except:
+            logger.exception("Ooops, query was not executed. "

```

Comment:
Please, add message of error to log

Reply:
Logger.exception(msg, *args, **kwargs)
Logs a message with level ERROR on this logger. The arguments are interpreted as for debug(), except that any passed exc_info is not inspected. Exception info is always added to the logging message. This method should only be called from an exception handler.

Reply:
+5